src/main/java/de/hpi/bpt/argos/api/eventType/EventTypeEndpointImpl.java

```diff
@@ -70,12 +70,14 @@ public String getEventTypes(Request request, Response response) {
         endpointUtil.logReceivedRequest(logger, request);
 
         List<EventType> eventTypes = PersistenceAdapterImpl.getInstance().getEventTypes();
-        JsonArray jsonEventTypes = new JsonArray();
-        // TODO where does the key for each event type come from? ("EventType{...}")
+        JsonObject jsonEventTypes = new JsonObject();

```

Comment:
It looked like it because of the name in front of the array. But alright, then I got it wrong.